src/endianconv.h

```diff
@@ -18,24 +18,15 @@
 #ifndef ENDIANCONV_H
 #define ENDIANCONV_H
 
-#include "config.h"
-#include <inttypes.h>
-
-#if defined(ENDIAN_LITTLE)
-
-/* No bswap16 for x86 */
-#define dcpu16(x) (uint16_t)__builtin_bswap32((x)<<16)
-#define dcpu32(x) __builtin_bswap32(x)
-#define dcpu64(x) __builtin_bswap64(x)
-
-#elif defined(ENDIAN_BIG)
-
-#define dcpu16(x) (x)
-#define dcpu32(x) (x)
-#define dcpu64(x) (x)
-
+#ifdef _WIN32
+#include <winsock.h>
```

Comment:
It looks like `stdlib.h` on MSVC implements functionality like the builtins: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/byteswap-uint64-byteswap-ulong-byteswap-ushort?view=vs-2019

So this could be implemented without `winsock.h`, which makes some other workarounds unnecessary: https://stackoverflow.com/a/32210953

Reply:
It'll break once libusb switches to winsock2. Pulling in a deprecated (?) header isn't really future proof